src/frontend/test/validation.js

```diff
@@ -293,7 +293,8 @@ const expandCardValidate = (field) => {
       field.scrollIntoView()
       $(this).off('shown.bs.collapse.foobar');
     })
-    $collapse.collapse('show')
+    if($collapse && $collapse.collapse)
```

Comment:
Testing broke so I had to add this - it is only in the "test" stub - the original remains unchanged